src/components/pages/ParentDash/RenderParentDash.js

```diff
@@ -1,25 +1,22 @@
-import React, { useEffect, useState, useRef } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Switch } from 'react-router-dom';
 import { SecureRoute } from '@okta/okta-react';
 import { Layout } from 'antd';
 import { ParentNav, DashHome, ChildSignup, Help, Logout } from './components';
 import { getParentDash } from '../../../api';
 
-import { useSetRecoilState, useRecoilValue } from 'recoil';
+import { useSetRecoilState, useRecoilState } from 'recoil';
 import { headerTitle } from '../../../state/headerTitle';
-import { userState } from '../../../state/userState';
+import { currentUserState } from '../../../state/userState';
 
 // ParentDash component that contains a nav bar and routes to the various components
-const RenderParentDash = props => {
+const RenderParentDash = () => {
   const setHeaderTitle = useSetRecoilState(headerTitle);
-
-  // Keeps track of the state for ParentNav
-
   const [userInfo, setUserInfo] = useState(null);
+  const [currentUser] = useRecoilState(currentUserState);
```

Comment:
This looks like it could be changed into using `useRecoilValue` and destructure the props you need to use.
Like this:
```js
const { curUserId, curUserToken } = useRecoilValue(currentUserState);
```